tomcat/datadog_checks/tomcat/data/metrics.yaml

```diff
@@ -46,13 +46,24 @@ jmx_metrics:
             alias: tomcat.servlet.request_count
             metric_type: counter
     - include:
-        type: Cache
```

Comment:
Are we sure `Cache` is not an outdated metric?

Reply:
It seems part of tomcat 7.0: https://github.com/apache/tomcat/blob/7.0.x/java/org/apache/catalina/core/StandardContext.java#L5293-L5297

> The Apache Tomcat team announces that support for Apache Tomcat 7.0.x will end on 31 March 2021.

https://tomcat.apache.org/tomcat-70-eol.html

So, it's safer to keep it for now. Will revert that change.